vllm/utils.py

```diff
@@ -693,3 +693,38 @@ def inner(*args, **kwargs):
         return inner  # type: ignore
 
     return wrapper
+
+
+@lru_cache(maxsize=5)
```

Comment:
why maxsize here? maybe just infinite size cache?

Reply:
I don't think letting any cache become unbound is a good practice :)